### PR TITLE
[TC-7000] Use UTC for maintenance window

### PR DESCRIPTION
**Pull request by roy-tc**
## Description

https://tradecloud.atlassian.net/browse/TC-7000

### Scope
This PR changes the time zone of the maintenance window to UTC.

### Submitter pre-flight check

- [ ] Review your documentation
- [ ] Add labels `needs reviewing`
- [ ] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ If you need any support either
 * send an email with your question to [support@tradecloud1.com](mailto:support@tradecloud.nl)
 * using the Intercom widget on the bottom right hand corner of the Tradecloud portal \(log in first\).
 
-API support is best effort, usually you will receive a response within 2 hours with a maximum of 24 hours on working hours in the CET timezone.
```
Comment (marcmatt) [inline support.md:14]: We do no support in the summer any more?

File: api/environments.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ It has an availability Service Level Objective of 95% per month.
 
 ## Planned production maintenance
 
-Maintenance can be planned on working days from 20:00 to 0:00 CE\(S\)T or in weekends and will be announced at least 1 working day ahead on the [Tradecloud status page](http://status.tradecloud1.com)
+Maintenance can be planned on working days from 19:00 to 23:00 UTC or in weekends and will be announced at least 1 working day ahead on the [Tradecloud status page](http://status.tradecloud1.com)
```
Comment (marcmatt) [inline api/environments.md:22]: I asked to add CET and CEST also, to not confuse the most of our users

File: support.md (hunk 1 above)
Comment (roy-tc) [reply support.md:14]: We do, that's why it has an `(S)` now in it.

File: api/environments.md (hunk 2 above)
Comment (roy-tc) [reply api/environments.md:22]: Honest mistake @marcmatt , can you please ammend in the Gitbooks (can be without PR). This ping-ponging feels inefficient

File: support.md (hunk 1 above)
Comment (marcmatt) [reply support.md:14]: Wauw, my mistake, I have to read from left to right

Comment (marcmatt) [reply support.md:14]: And cancel my headache

**Pull request merged**